kubernetes/deployment-production.tmpl

```diff
@@ -344,6 +253,109 @@ spec:
 ---
 apiVersion: apps/v1
 kind: Deployment
+metadata:
+  name: panoptes-production-sidekiq
+  labels:
+    app: panoptes-production-sidekiq
+spec:
+  selector:
+    matchLabels:
+      app: panoptes-production-sidekiq
+  template:
+    metadata:
+      labels:
+        app: panoptes-production-sidekiq
+    spec:
+      containers:
+        - name: panoptes-production-sidekiq
+          image: zooniverse/panoptes:__IMAGE_TAG__
+          resources:
+            requests:
+              memory: "1500Mi"
+              cpu: "500m"
+              ephemeral-storage: "50Gi"
+            limits:
+              memory: "1500Mi"
+              cpu: "1000m"
+              ephemeral-storage: "50Gi"
```

Comment:
```suggestion
              memory: "4000Mi"
              cpu: "1000m"
              ephemeral-storage: "50Gi"
            limits:
              memory: "4000Mi"
              cpu: "2000m"
              ephemeral-storage: "50Gi"
```
aws ec2 metrics suggest this bursts ~3.25GB, 4GB might be overkill but i'd like to avoid OOMKiller on these.

Assuming we have bigger node types CPU can burst to 2vcpus - aws metrics shows ~75% of 2vcpus most of the time.